imgproc.yml

```
@@ -4,13 +4,15 @@ channels:
   - nodefaults
 dependencies:
   - conda-forge::python==3.7
-  - conda-forge::matplotlib==3.5.3
+  - conda-forge::matplotlib==3.3.2
   - conda-forge::scikit-image==0.19.3
   - conda-forge::numpy==1.21.6
   - conda-forge::ipykernel==6.16.2
   - conda-forge::pandas==1.3.5 # a dependency of pylattice
   - conda-forge::pillow==9.2.0 # a dependency of pylattice
   - conda-forge::pims==0.6.1 # a dependency of pylattice
-  - conda-forge::pip
+  - conda-forge::pip==23.0.1
+  - conda-forge::python-igraph==0.7.1.post7 # a dependency of MitoTNT
   - pip:
     - pylattice # dependencies are not installed automatically but are listed in https://github.com/pylattice/pyLattice/blob/master/requirements.txt
```

```suggestion
    - pylattice==0.1.3  # dependencies are not installed automatically but are listed in https://github.com/pylattice/pyLattice/blob/master/requirements.txt
```
sorry, there's just one more suggestion! I forgot to add this before, but we should probably pin pylattice, as well